wiki/faq.md

```diff
@@ -233,9 +233,25 @@ only.
 ---
 ### What happens if I stake on a fork?
 
-When you stake on a fork, the stake you see is not actually legitimate. When your wallet figures out that it is on a fork, you will see these rewards disappear as the wallet corrects itself. This process happens automatically. While frustrating, once off of the fork, you are still able to stake again. 
+When you stake on a [fork](forks "wikilink"), the stake you see is not actually legitimate. When your wallet figures out that it is on a fork, you will see these rewards disappear as the wallet corrects itself. This process happens automatically. While frustrating, once off of the fork, you are still able to stake again. 
 
 ---
+
+### My pending rewards are stuck at 16384 GRC
+
+If you see your pending rewards at 16384 GRC, you have hit the maximum amount

```

Comment:
```suggestion
If you see your pending rewards at 16384 GRC, you have hit the maximum amount of

```

Reply:
you put "ofs"?

Reply:
Whoops. Not even sure how I did that since I just clicked "add suggestion to batch" here